test-cases/performance/perf-regression-predefined-throughput-steps.yaml

```diff
@@ -20,13 +20,28 @@ stress_cmd_r: [
   "cassandra-stress read no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000"
 ]
 
+stress_cmd_r_warm_up: [
+  "cassandra-stress read no-warmup cl=ALL n=5000000 -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
```

Comment:
Why are you using “no-warmup” in the command here and everywhere else.
It wasn’t part of the commands I gave you.

Reply:
We always ran the warm-up as: in `stress_cmd_r` commands: changed  CL and replaced duration and threads. So this is what I did here.

Reply:
removed